src/legacy/core_plugins/kibana/public/discover/_index.scss

```diff
@@ -23,4 +23,4 @@
 @import 'doc_viewer/index';
 
 // Context styles
-@import 'context/index';
+@import 'angular/context/index';
```

Comment:
Dear @elastic/kibana-design , valued owner and ruler of the 7 SCSS code kingdoms
There were just path changes in this PR.
Your humble servant
@kertal